src/views/ModelPlan/ReadOnly/_components/FilterView/BodyContent/index.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { Alert, Grid, Link } from '@trussworks/react-uswds';
+
+import ReadOnlyBeneficiaries from 'views/ModelPlan/ReadOnly/Beneficiaries';
+import ReadOnlyGeneralCharacteristics from 'views/ModelPlan/ReadOnly/GeneralCharacteristics';
+import ReadOnlyModelBasics from 'views/ModelPlan/ReadOnly/ModelBasics';
+import ReadOnlyOpsEvalAndLearning from 'views/ModelPlan/ReadOnly/OpsEvalAndLearning';
+import ReadOnlyParticipantsAndProviders from 'views/ModelPlan/ReadOnly/ParticipantsAndProviders';
+import ReadOnlyPayments from 'views/ModelPlan/ReadOnly/Payments';
+import ReadOnlyTeamInfo from 'views/ModelPlan/ReadOnly/Team';
+
+import allPossibleFilterViews from './_filterGroupMapping';
+
+const FitleredViewSection = ({
+  children,
+  lastSection,
+  sectionName
+}: {
+  children: React.ReactNode;
+  lastSection?: boolean;
+  sectionName: string;
+}) => {
+  return (
+    <div
+      className={`filtered-view-section filtered-view-section--${sectionName} margin-bottom-6 ${
+        !lastSection
+          ? 'border-bottom border-base-light padding-bottom-4'
+          : 'padding-bottom-0'
+      }`}
+    >
+      {children}
+    </div>
+  );
+};
+
+const BodyContent = ({
+  modelID,
+  filteredView
+}: {
+  modelID: string;
+  filteredView: string;
+}) => {
+  const { t } = useTranslation('filterView');
+
+  const individualFilterView =
+    allPossibleFilterViews[filteredView as keyof typeof allPossibleFilterViews];
+
+  return (
+    <Grid>
+      <FitleredViewSection sectionName="model-team">
+        <h2 className="margin-top-0 margin-bottom-4">Model Team</h2>
+        <ReadOnlyTeamInfo modelID={modelID} isViewingFilteredView />
+      </FitleredViewSection>
+
+      {Object.keys(individualFilterView).map(task => {
+        if (task === 'basics') {
+          return (
+            <FitleredViewSection
+              sectionName="model-basics"
+              key={task}
+              lastSection={task === Object.keys(individualFilterView).pop()}
+            >
+              <ReadOnlyModelBasics
+                modelID={modelID}
+                filteredView={filteredView}
+                isViewingFilteredView
+                filteredQuestions={
+                  individualFilterView[
+                    task as keyof typeof individualFilterView
+                  ]
+                }
+              />
+            </FitleredViewSection>
+          );
+        }
+
```

Comment:
I think all of this can be dynamic.  Something like this would work without needing to explicitly define each elements props as they are all almost identical

```
const components: Record<string, React.ElementType> = {
  basics: ReadOnlyModelBasics,
  'general-characteristics': ReadOnlyGeneralCharacteristics,
  'participants-and-providers': ReadOnlyParticipantsAndProviders,
  beneficiaries: ReadOnlyBeneficiaries,
  'ops-eval-and-learning': ReadOnlyOpsEvalAndLearning,
  payments: ReadOnlyPayments
};

{Object.keys(individualFilterView).map(task => {
        const FitleredViewSectionContent = components[task];

        return (
          <FitleredViewSection
            sectionName={task}
            key={task}
            lastSection={task === Object.keys(individualFilterView).pop()}
          >
            <FitleredViewSectionContent
              modelID={modelID}
              filteredView={filteredView}
              isViewingFilteredView
              filteredQuestions={
                individualFilterView[task as keyof typeof individualFilterView]
              }
            />
          </FitleredViewSection>
        );
})
```